src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
I think Union type fits better here, because you know for sure which values can take shape and color